layouts/partials/settings.html

```diff
@@ -1,6 +1,18 @@
 {{- range $i, $def := (getJSON (printf "themes/tvpage/defaults/%s.json" .Type)).option -}}
 {{- $.Scratch.SetInMap "settings" .code .value -}}
 {{- end -}}
+{{- $type := .Type -}}
+{{- range readDir "/" -}}
+	{{- if eq .Name "defaults" -}}
+		{{- $directory := "defaults/" -}}
+		{{- $filename := (printf "%s.json" $type) -}}
+		{{- if (where (readDir $directory) "Name" $filename) -}}
+			{{- range $i, $def := (getJSON (printf "defaults/%s.json" $type)).option -}}
+			{{- $.Scratch.SetInMap "settings" .code .value -}}
```

Comment:
Check if you can use range in the simplest version (without  $i, $def :=) since these variables are not being used.